### ISSUE TITLE
Steps in features files are marked as not implemented, but they still exists

### ISSUE THREAD

**Issue by boris779** [cucumber/cucumber-eclipse#159]
Hi!

I have some steps which already have a glue code (step definition) but they are marked as not implemented in the feature file by the cucumber plugin. It seems that the plugin cannot handle my regular expression.

Cucumber itself finds the implementation when execute it.

Note: I use cucumber in german and these steps uses a datatable-value

feature:
Angenommen es existiert der Hof <Hofname> mit WE-Nummer <WE-Nr> und folgenden Adressen

@Angenommen("^es existiert der Hof (.+) mit WE-Nummer ([0-9]{2}-[0-9]{2},[0-9]{3}) und folgenden Adressen$")

![2016-04-20_11h09_01](https://cloud.githubusercontent.com/assets/10259016/14669272/67d1a8ba-06e8-11e6-976c-02c002db4268.png)
![2016-04-20_11h10_32](https://cloud.githubusercontent.com/assets/10259016/14669311/84522c26-06e8-11e6-8166-7d43c8fd3db6.png)

**Comment by LorenzoBettini** [comment]
Looks like the problem is still there: after creating the steps with the quickfix the warnings are still there... the only way I found to make the warnings go away is to close the project and open it again... does it store some cache somewhere which is not correctly updated?

**Comment by girijant** [comment]
@LorenzoBettini, 
Instead of closing project, Only close the opened feature file from eclipse editor and reopen it. Also remove unused import statements from step definition file by ctrl+shift+O.

**Comment by LorenzoBettini** [comment]
Right, closing only the feature editor works. Still this shouldn't be required to make the warnings go away :-)

**Comment by mgor** [comment]
@boris779, @LorenzoBettini, this issue is solved with PR #240, which is merged now. Could you clone the project, run `mvn clean install` and then update/install the plugin from archive `cucumber.eclipse.download/download/0.0.20-SNAPSHOT/cucumber-eclipse-plugin-0.0.20-SNAPSHOT.zip`?

Also make sure that your `.project` file contains both the cucumber nature:
```xml
<nature>cucumber.eclipse.steps.jdt.stepsNature</nature>
``` 
and the build command:
```xml
<buildCommand>
  <name>cucumber.eclipse.steps.jdt.stepsBuilder</name>
  <arguments>
  </arguments>
</buildCommand>
```

**Comment by girijant** [comment]
@boris779, @LorenzoBettini, @mgor, 
Currently I am fixing some major defects. I will fire a build soon with a new version.

Thanks, 
Girija

**Comment by girijant** [comment]
@boris779, @LorenzoBettini, @mgor,
Released 0.0.21-SNAPSHOT with some of major bgg fixes and Pull-Requests #240 and #239. 
Please updating your existing plugin and retry.

Thanks,
-Girija

**Comment by boris779** [comment]
Sorry for the late response.
